v20.1/set-vars.md

```diff
@@ -166,7 +166,7 @@ The following demonstrates how to assign a list of values:
 
 {{site.data.alerts.callout_danger}}As a best practice, we recommend not using this setting and avoid setting a session time for your database. We instead recommend converting UTC values to the appropriate time zone on the client side.{{site.data.alerts.end}}
 
-You can control your client's default time zone for the current session with <code>SET TIME ZONE</code>. This will apply a session offset to all [`TIMESTAMP WITH TIME ZONE`](timestamp.html) values.
+You can control your client's default time zone for the current session with <code>SET TIME ZONE</code>. This will apply a session offset to all [`TIMESTAMPTZ`\`TIMESTAMP WITH TIME ZONE`](timestamp.html) and [`TIMETZ`\`TIME WITH TIME ZONE`](time.html) values.
```

Comment:
`\` is not being rendered properly. Change `\` with "or".
Suggested change: [`TIMESTAMPTZ` or `TIMESTAMP WITH TIME ZONE`] and
 [`TIMETZ` or `TIME WITH TIME ZONE`]